docs/en/structured-data/transformation-service.md

```diff
@@ -6,11 +6,61 @@ group: Structured Data
 priority: 1
 ---
 
-* Table of Contents
-{:toc}
-
 ## Data Transformations
 
+With job service and catalog service, Alluxio can transform a table 
+to a new table at partition level. The data of the original table is not modified,
```

Comment:
new sentences should be on a new line:

https://docs.alluxio.io/os/user/stable/en/contributor/Documentation-Conventions.html#line-breaks